src/server/common/controller/question-page-controller/question-page-controller-redirect-override.test.js

```diff
@@ -0,0 +1,159 @@
+import { AnswerModel } from '../../model/answer/answer-model.js'
+import { ExitPage } from '../../model/page/exit-page-model.js'
+import { QuestionPage } from '../../model/page/question-page-model.js'
+import { createServer } from '~/src/server/index.js'
+import SessionTester from '../../test-helpers/session-helper.js'
+import { QuestionPageController } from './question-page-controller.js'
+import { withCsrfProtection } from '../../test-helpers/csrf.js'
+import { statusCodes } from '../../constants/status-codes.js'
+
+/** @import { Server } from '@hapi/hapi' */
+
+const questionUrl = '/dummy/question-url'
+const nextQuestionUrl = '/dummy/next-question-url'
+const overriddenQuestionUrl = '/dummy/overridden-question-url'
+
+const sectionKey = 'dummy'
+const questionKey = 'question'
+
+class TestAnswer extends AnswerModel {
+  toState() {
+    return this._data?.questionKey
+  }
+
+  static fromState(state) {
+    return new TestAnswer(
+      state !== undefined ? { questionKey: state } : undefined
+    )
+  }
+
+  get value() {
+    return this._data?.[questionKey]
+  }
+
+  _extractFields(data) {
+    return {
+      [questionKey]: data[questionKey]
+    }
+  }
+
+  validate() {
+    return { isValid: true, errors: {} }
+  }
+}
+
+class TestExitPage extends ExitPage {
+  urlPath = '/exit'
+}
+
+class TestPage extends QuestionPage {
+  question = 'dummy question'
+  questionKey = questionKey
+  view = 'some file'
+  sectionKey = sectionKey
+  urlPath = questionUrl
+
+  Answer = TestAnswer
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can remove this and change `_answer` to `answer`

Reply:
removing this breaks the tests

Reply:
I'm a bit confused - I don't mean removing the param.  I mean removing the eslint - because we do use the var, so there's no need to tell eslint not to complain?

And since we use the var, `_answer` -> `answer` would be the normal convention.

Reply:
oh i see - miss understood what you meant